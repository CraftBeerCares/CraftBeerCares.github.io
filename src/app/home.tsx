import Image from "next/image";
import Link from "next/link";
import styles from "./home.module.css";
import { secondary } from "@/styles/fonts";
import { clsx } from "clsx";
import { PageTemplate, ArticleRow, Testimonials } from "@/components";

export const Home = () => {
  return (
    <PageTemplate>
      <div className={styles.logoContainer}>
        <Image src="/assets/logo.svg" alt="Craft Beer Cares Logo" fill={true} />
      </div>

      <h1 className={clsx(secondary.className, styles.subTitle)}>
        Craft Beer Cares is a collective of beer lovers who want to do good
        things.
      </h1>

      <ArticleRow imageSrc="/assets/001.webp" imageAlt="Image of Founder">
        <p>
          We take part in events run by amazing people with a passion for
          fundraising, and have organised events combining charitable aims with
          the enjoyment of beer made by some of the most talented breweries in
          the UK and beyond.
        </p>
        <p>
          Our events have raised funds for charities such as Art Against Knives,
          Hackney Winter Night Shelter, Mind, The Newman Holiday Trust and more!
        </p>
      </ArticleRow>

      <section className={styles.linkContainer}>
        <Link
          className={clsx(secondary.className, styles.navLink)}
          href="/events"
        >
          Check out The Events We&apos;ve Done For charity
        </Link>
      </section>

      <ArticleRow
        imageSrc="/assets/002.webp"
        imageAlt="DJ at Craft Beers Event"
        imagePosition="right"
      >
        <p>
          Not only do we put on events, but we also shout about events that
          great pubs, bars, breweries, and other oganisations are doing to
          benefit good causes.
        </p>
        <p>
          Find out more about the events we organise and other good things
          happening by signing up to our newsletter, or by following us on
          Twitter, Facebook, or Instagram.
        </p>
      </ArticleRow>

      <section>
        <h1 className={clsx(secondary.className, styles.quoteTitle)}>
          Here&apos;s what some nice folks have said about craft beer cares!
        </h1>
      </section>

      <section className={styles.quoteContainer}>
        <Testimonials />
      </section>
    </PageTemplate>
  );
};
