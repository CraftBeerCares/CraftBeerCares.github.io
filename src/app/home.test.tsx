import { render } from "@testing-library/react";
import { Home } from "./home";

it("renders Home page", () => {
  const { container } = render(<Home />);
  expect(container).toMatchSnapshot();
});
