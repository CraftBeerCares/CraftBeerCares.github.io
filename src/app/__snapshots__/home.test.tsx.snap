// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Home page 1`] = `
<div>
  <div
    class="pageContainer"
  >
    <div
      class="header"
    >
      <div
        class="content"
      >
        <header>
          <div
            class="header"
          >
            <button
              class="burgerMenuButton"
            >
              <img
                alt="Burger Menu"
                data-nimg="fill"
                decoding="async"
                loading="lazy"
                src="/assets/menu.svg"
                style="position: absolute; height: 100%; width: 100%; left: 0px; top: 0px; right: 0px; bottom: 0px; object-fit: contain; color: transparent; filter: invert(1);"
              />
            </button>
            <h1
              class="className title"
            >
              Craft Beer Cares
            </h1>
            <nav
              class="topNav className"
            >
              <a
                class="navLink"
                href="/"
              >
                Home
              </a>
              <a
                class="navLink"
                href="/about"
              >
                About
              </a>
              <a
                class="navLink"
                href="/join_us"
              >
                Join Us
              </a>
              <a
                class="navLink"
                href="/events"
              >
                Events
              </a>
              <a
                class="navLink"
                href="/shop"
              >
                Shop
              </a>
              <a
                class="navLink"
                href="/contact"
              >
                Contact
              </a>
            </nav>
          </div>
        </header>
      </div>
    </div>
    <div
      class="scrollContainer"
    >
      <main
        class="className content"
      >
        <div
          class="logoContainer"
        >
          <img
            alt="Craft Beer Cares Logo"
            data-nimg="fill"
            decoding="async"
            loading="lazy"
            src="/assets/logo.svg"
            style="position: absolute; height: 100%; width: 100%; left: 0px; top: 0px; right: 0px; bottom: 0px; color: transparent;"
          />
        </div>
        <h1
          class="className subTitle"
        >
          Craft Beer Cares is a collective of beer lovers who want to do good things.
        </h1>
        <div
          class="articleRow"
        >
          <div
            class="articleImage imageLeft"
          >
            <img
              alt="Image of Founder"
              data-nimg="fill"
              decoding="async"
              loading="lazy"
              sizes="100vw"
              src="/_next/image?url=%2Fassets%2F001.webp&w=3840&q=75"
              srcset="/_next/image?url=%2Fassets%2F001.webp&w=640&q=75 640w, /_next/image?url=%2Fassets%2F001.webp&w=750&q=75 750w, /_next/image?url=%2Fassets%2F001.webp&w=828&q=75 828w, /_next/image?url=%2Fassets%2F001.webp&w=1080&q=75 1080w, /_next/image?url=%2Fassets%2F001.webp&w=1200&q=75 1200w, /_next/image?url=%2Fassets%2F001.webp&w=1920&q=75 1920w, /_next/image?url=%2Fassets%2F001.webp&w=2048&q=75 2048w, /_next/image?url=%2Fassets%2F001.webp&w=3840&q=75 3840w"
              style="position: absolute; height: 100%; width: 100%; left: 0px; top: 0px; right: 0px; bottom: 0px; object-fit: contain; color: transparent;"
            />
          </div>
          <article
            class="article articleRight"
          >
            <p>
              We take part in events run by amazing people with a passion for fundraising, and have organised events combining charitable aims with the enjoyment of beer made by some of the most talented breweries in the UK and beyond.
            </p>
            <p>
              Our events have raised funds for charities such as Art Against Knives, Hackney Winter Night Shelter, Mind, The Newman Holiday Trust and more!
            </p>
          </article>
        </div>
        <section
          class="linkContainer"
        >
          <a
            class="className navLink"
            href="/events"
          >
            Check out The Events We've Done For charity
          </a>
        </section>
        <div
          class="articleRow"
        >
          <div
            class="articleImage imageRight"
          >
            <img
              alt="DJ at Craft Beers Event"
              data-nimg="fill"
              decoding="async"
              loading="lazy"
              sizes="100vw"
              src="/_next/image?url=%2Fassets%2F002.webp&w=3840&q=75"
              srcset="/_next/image?url=%2Fassets%2F002.webp&w=640&q=75 640w, /_next/image?url=%2Fassets%2F002.webp&w=750&q=75 750w, /_next/image?url=%2Fassets%2F002.webp&w=828&q=75 828w, /_next/image?url=%2Fassets%2F002.webp&w=1080&q=75 1080w, /_next/image?url=%2Fassets%2F002.webp&w=1200&q=75 1200w, /_next/image?url=%2Fassets%2F002.webp&w=1920&q=75 1920w, /_next/image?url=%2Fassets%2F002.webp&w=2048&q=75 2048w, /_next/image?url=%2Fassets%2F002.webp&w=3840&q=75 3840w"
              style="position: absolute; height: 100%; width: 100%; left: 0px; top: 0px; right: 0px; bottom: 0px; object-fit: contain; color: transparent;"
            />
          </div>
          <article
            class="article articleLeft"
          >
            <p>
              Not only do we put on events, but we also shout about events that great pubs, bars, breweries, and other oganisations are doing to benefit good causes.
            </p>
            <p>
              Find out more about the events we organise and other good things happening by signing up to our newsletter, or by following us on Twitter, Facebook, or Instagram.
            </p>
          </article>
        </div>
        <section>
          <h1
            class="className quoteTitle"
          >
            Here's what some nice folks have said about craft beer cares!
          </h1>
        </section>
        <section
          class="quoteContainer"
        >
          <div
            class="carousel"
          >
            <div
              aria-label="slider"
              aria-live="polite"
              class="horizontalSlider___281Ls carousel__slider carousel__slider--horizontal"
              role="listbox"
              tabindex="0"
            >
              <div
                class="carousel__slider-tray-wrapper carousel__slider-tray-wrap--horizontal"
              >
                <div
                  class="sliderTray___-vHFQ sliderAnimation___300FY carousel__slider-tray carousel__slider-tray--horizontal"
                  style="display: flex; align-items: stretch; width: 500%; transform: translateX(0%) translateX(0px); flex-direction: row;"
                >
                  <div
                    aria-label="slide"
                    aria-selected="true"
                    class="slide___3-Nqo slideHorizontal___1NzNV carousel__slide carousel__slide--visible quoteBox"
                    role="option"
                    style="width: 20%;"
                    tabindex="0"
                  >
                    <div
                      class="slideInner___2mfX9 carousel__inner-slide"
                      style="position: unset;"
                    >
                      <div
                        class="quoteBox"
                      >
                        <img
                          alt="Quote Start"
                          class="quoteImage"
                          data-nimg="1"
                          decoding="async"
                          height="24"
                          loading="lazy"
                          src="/assets/quote_start.svg"
                          style="color: transparent;"
                          width="24"
                        />
                        <p
                          class="quote"
                        >
                          Craft Beer Cares were one of the first UK groups to create a space for craft beer lovers to drink for a cause, and their charitable work is hugely important to the industry and the organisations they support. G knows how to throw a great party, and I'm looking forward to attending more CBC fundraisers in the future!
                        </p>
                        <img
                          alt="Quote End"
                          class="quoteImage"
                          data-nimg="1"
                          decoding="async"
                          height="24"
                          loading="lazy"
                          src="/assets/quote_end.svg"
                          style="color: transparent;"
                          width="24"
                        />
                        <p
                          class="author"
                        >
                          Ruvani de Silva
                        </p>
                      </div>
                    </div>
                  </div>
                  <div
                    aria-label="slide"
                    aria-selected="false"
                    class="slide___3-Nqo slideHorizontal___1NzNV carousel__slide carousel__slide--hidden quoteBox"
                    role="option"
                    style="width: 20%;"
                    tabindex="-1"
                  >
                    <div
                      class="slideInner___2mfX9 carousel__inner-slide"
                      style="position: unset;"
                    >
                      <div
                        class="quoteBox"
                      >
                        <img
                          alt="Quote Start"
                          class="quoteImage"
                          data-nimg="1"
                          decoding="async"
                          height="24"
                          loading="lazy"
                          src="/assets/quote_start.svg"
                          style="color: transparent;"
                          width="24"
                        />
                        <p
                          class="quote"
                        >
                          Craft Beer Cares tightly knits the best aspects of the beer industry: excellent beer and excellent community in a super fun festival format that also does some good. What is there not to love about that?
                        </p>
                        <img
                          alt="Quote End"
                          class="quoteImage"
                          data-nimg="1"
                          decoding="async"
                          height="24"
                          loading="lazy"
                          src="/assets/quote_end.svg"
                          style="color: transparent;"
                          width="24"
                        />
                        <p
                          class="author"
                        >
                          Natasha W.
                        </p>
                      </div>
                    </div>
                  </div>
                  <div
                    aria-label="slide"
                    aria-selected="false"
                    class="slide___3-Nqo slideHorizontal___1NzNV carousel__slide carousel__slide--hidden quoteBox"
                    role="option"
                    style="width: 20%;"
                    tabindex="-1"
                  >
                    <div
                      class="slideInner___2mfX9 carousel__inner-slide"
                      style="position: unset;"
                    >
                      <div
                        class="quoteBox"
                      >
                        <img
                          alt="Quote Start"
                          class="quoteImage"
                          data-nimg="1"
                          decoding="async"
                          height="24"
                          loading="lazy"
                          src="/assets/quote_start.svg"
                          style="color: transparent;"
                          width="24"
                        />
                        <p
                          class="quote"
                        >
                          I'm a CBC veteran volunteer - And I can't wait for the next festival. It's the friendliest, fun-est crowd and the volunteers are an amazing bunch of enthusiastic, knowledgeable, efficient and kind people who make the whole event an all day party.
                        </p>
                        <img
                          alt="Quote End"
                          class="quoteImage"
                          data-nimg="1"
                          decoding="async"
                          height="24"
                          loading="lazy"
                          src="/assets/quote_end.svg"
                          style="color: transparent;"
                          width="24"
                        />
                        <p
                          class="author"
                        >
                          Mona B.
                        </p>
                      </div>
                    </div>
                  </div>
                  <div
                    aria-label="slide"
                    aria-selected="false"
                    class="slide___3-Nqo slideHorizontal___1NzNV carousel__slide carousel__slide--hidden quoteBox"
                    role="option"
                    style="width: 20%;"
                    tabindex="-1"
                  >
                    <div
                      class="slideInner___2mfX9 carousel__inner-slide"
                      style="position: unset;"
                    >
                      <div
                        class="quoteBox"
                      >
                        <img
                          alt="Quote Start"
                          class="quoteImage"
                          data-nimg="1"
                          decoding="async"
                          height="24"
                          loading="lazy"
                          src="/assets/quote_start.svg"
                          style="color: transparent;"
                          width="24"
                        />
                        <p
                          class="quote"
                        >
                          What impressed me most was the collegial and friendly environment, among the staff but also between us and the punters, who all seemed to understand that this wasn't your usual festival.
                        </p>
                        <img
                          alt="Quote End"
                          class="quoteImage"
                          data-nimg="1"
                          decoding="async"
                          height="24"
                          loading="lazy"
                          src="/assets/quote_end.svg"
                          style="color: transparent;"
                          width="24"
                        />
                        <p
                          class="author"
                        >
                          Ewan M.
                        </p>
                      </div>
                    </div>
                  </div>
                  <div
                    aria-label="slide"
                    aria-selected="false"
                    class="slide___3-Nqo slideHorizontal___1NzNV carousel__slide carousel__slide--hidden quoteBox"
                    role="option"
                    style="width: 20%;"
                    tabindex="-1"
                  >
                    <div
                      class="slideInner___2mfX9 carousel__inner-slide"
                      style="position: unset;"
                    >
                      <div
                        class="quoteBox"
                      >
                        <img
                          alt="Quote Start"
                          class="quoteImage"
                          data-nimg="1"
                          decoding="async"
                          height="24"
                          loading="lazy"
                          src="/assets/quote_start.svg"
                          style="color: transparent;"
                          width="24"
                        />
                        <p
                          class="quote"
                        >
                          Had great fun attending the festival for the first time some years ago when it was held at Brew Club and such an honourable experience having been able to bring in my swissrolls at another festival in 2019!
                        </p>
                        <img
                          alt="Quote End"
                          class="quoteImage"
                          data-nimg="1"
                          decoding="async"
                          height="24"
                          loading="lazy"
                          src="/assets/quote_end.svg"
                          style="color: transparent;"
                          width="24"
                        />
                        <p
                          class="author"
                        >
                          Shima
                        </p>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
            <div
              class="controls"
            >
              <button
                aria-label="previous"
                class="buttonBack___1mlaL carousel__back-button button"
                type="button"
              >
                Back
              </button>
              <div
                class="carousel__dot-group dots"
              >
                <button
                  aria-label="slide dot"
                  class="dot___3c3SI carousel__dot carousel__dot--0 carousel__dot--selected"
                  disabled=""
                  type="button"
                >
                  <span />
                </button>
                <button
                  aria-label="slide dot"
                  class="dot___3c3SI carousel__dot carousel__dot--1"
                  type="button"
                >
                  <span />
                </button>
                <button
                  aria-label="slide dot"
                  class="dot___3c3SI carousel__dot carousel__dot--2"
                  type="button"
                >
                  <span />
                </button>
                <button
                  aria-label="slide dot"
                  class="dot___3c3SI carousel__dot carousel__dot--3"
                  type="button"
                >
                  <span />
                </button>
                <button
                  aria-label="slide dot"
                  class="dot___3c3SI carousel__dot carousel__dot--4"
                  type="button"
                >
                  <span />
                </button>
              </div>
              <button
                aria-label="next"
                class="buttonNext___2mOCa carousel__next-button button"
                type="button"
              >
                Next
              </button>
            </div>
          </div>
        </section>
      </main>
      <div
        class="footer"
      >
        <div
          class="content"
        >
          <footer
            class="footer"
          >
            <div
              class="footerMobile"
            >
              <div
                class="footerLogo"
              >
                <img
                  alt="Craft Beer Cares Logo"
                  data-nimg="fill"
                  decoding="async"
                  loading="lazy"
                  src="/assets/logo_bw.svg"
                  style="position: absolute; height: 100%; width: 100%; left: 0px; top: 0px; right: 0px; bottom: 0px; color: transparent; filter: invert(1);"
                />
              </div>
            </div>
            <div
              class="footerDesktop"
            >
              <div
                class="footerColumn"
              >
                <h4
                  class="className title"
                >
                  Craft Beer Cares
                </h4>
                <nav
                  class="className footerLinks"
                >
                  <a
                    class="navLink"
                    href="/"
                  >
                    Home
                  </a>
                  <a
                    class="navLink"
                    href="/about"
                  >
                    About
                  </a>
                  <a
                    class="navLink"
                    href="/join_us"
                  >
                    Join Us
                  </a>
                  <a
                    class="navLink"
                    href="/events"
                  >
                    Events
                  </a>
                  <a
                    class="navLink"
                    href="/shop"
                  >
                    Shop
                  </a>
                  <a
                    class="navLink"
                    href="/contact"
                  >
                    Contact
                  </a>
                </nav>
                <div
                  class="socialLinks"
                >
                  <a
                    class="socialLink"
                    href="https://www.facebook.com/craftbeercares"
                  >
                    <img
                      alt="Facebook"
                      data-nimg="1"
                      decoding="async"
                      height="30"
                      loading="lazy"
                      src="/assets/facebook.svg"
                      style="color: transparent;"
                      width="30"
                    />
                  </a>
                  <a
                    class="socialLink"
                    href="https://www.instagram.com/craftbeercares/"
                  >
                    <img
                      alt="Instagram"
                      data-nimg="1"
                      decoding="async"
                      height="30"
                      loading="lazy"
                      src="/assets/instagram.svg"
                      style="color: transparent;"
                      width="30"
                    />
                  </a>
                  <a
                    class="socialLink"
                    href="https://twitter.com/craftbeercares"
                  >
                    <img
                      alt="Twitter"
                      data-nimg="1"
                      decoding="async"
                      height="30"
                      loading="lazy"
                      src="/assets/twitter.svg"
                      style="color: transparent;"
                      width="30"
                    />
                  </a>
                </div>
                <div
                  class="className footerQuote"
                >
                  Craft Beer Cares - Drinking beer for good Stuff since 2017
                </div>
              </div>
              <div
                class="footerColumn"
              >
                <div
                  class="footerLogo"
                >
                  <img
                    alt="Craft Beer Cares Logo"
                    data-nimg="fill"
                    decoding="async"
                    loading="lazy"
                    src="/assets/logo_bw.svg"
                    style="position: absolute; height: 100%; width: 100%; left: 0px; top: 0px; right: 0px; bottom: 0px; color: transparent; filter: invert(1);"
                  />
                </div>
              </div>
              <div
                class="footerColumn rightColumn"
              >
                <div
                  class="signUpBox"
                >
                  <h4
                    class="className title"
                  >
                    Sign Up for Our Monthlyish Newsletters
                  </h4>
                  <p
                    class="className signUpText"
                  >
                    Hear about upcoming events and initiatives from us and others pairing beer with charitable giving. We won’t share your info with anyone, and you can opt out at anytime.
                  </p>
                  <input
                    class="emailBox"
                    placeholder="Your e-mail"
                  />
                  <button
                    class="signUpButton"
                  >
                    Sign Up!
                  </button>
                </div>
              </div>
            </div>
          </footer>
        </div>
      </div>
    </div>
  </div>
</div>
`;
